src/lib/fast_check.py

```diff
@@ -96,6 +96,25 @@ async def get_dynatrace_token_metadata(dt_session: ClientSession, context: Loggi
         return {}
 
 
+async def check_dynatrace(logging_context: LoggingContext, project_id, dt_session: ClientSession, dynatrace_url, dynatrace_access_key):
+    try:
+
+        if not dynatrace_url or not dynatrace_access_key:
+            logging_context.log(f'ERROR No Dynatrace secrets: DYNATRACE_URL, DYNATRACE_ACCESS_KEY for project: {project_id}.'
+                                     f'Add required secrets to Secret Manager.')
+            return None
+        logging_context.log(f"Using [DYNATRACE_URL] Dynatrace endpoint: {dynatrace_url}")
+        logging_context.log(f'Using [DYNATRACE_ACCESS_KEY]: {dynatrace_access_key[0:dynatrace_access_key.rfind(".")]}.')
```

Comment:
Customers can still also use tokens in old format - this line will reveal the token.
Characters until first dot are usually the same in new format:  'dt0c01'